extensions/adobe/experience/implementations.schema.json

```diff
@@ -28,6 +28,11 @@
       "$ref": "https://ns.adobe.com/xdm/context/implementationdetails",
       "description":
         "Client details interacting with Adobe Experience Cloud ID Service."
+    },
+    "https://ns.adobe.com/experience/adcloud": {
+      "title": "Client Implementation for Adobe AdCloud",
```

Comment:
No references to specific Adobe products

Reply:
Should be ok in this context. We have a couple of other Adobe-specific references here, too.

Reply:
Didn't realize this was inside an extension schema - this is fine.

Reply:
I believe all concerns are now addressed from these comments.